app/helpers/cm_admin/application_helper.rb

```diff
@@ -7,5 +7,11 @@ module ApplicationHelper
     def current_webpacker_instance
       CmAdmin.webpacker
     end
+
+    # Allow if policy is defined.
```

Comment:
```suggestion
    # Allow if policy is not defined.
```

I'm not really sure if my suggestion is right, sorry not too fluent in gems :D